app/client/components/navBar/page.tsx

```diff
@@ -34,6 +40,16 @@ const Page: React.FunctionComponent<OuterProps> = props => {
 
   const [collapsed, setCollapsed] = React.useState(false);
 
+  const login: LoginState = useSelector((state: ReduxState) => state.login);
+  const { organisationIds } = login;
+  const multiOrgActive = organisationIds && organisationIds.length > 1;
+
+  if (multiOrgActive && !isMultiOrg) {
+    // Trying to access a page with multi org active
+    browserHistory.push("/");
+    message.error("This page is unsupported with multi organisation");
+  }
```

Comment:
protect views from being accessed when multi org is active and page isn't `isMultiOrg=true`. this protects users from going to a page directly in the URL